docs/reference/widgets.md

```diff
@@ -35,6 +35,71 @@ An input field for a single string value.
 
 - **disabled** `Boolean` _optional_: Define if the field should be displayed as disabled thus no input can be filled.
 
+## NumberInput
+
+An input field for numbers e.g integers, float etc.
+
+### Usage
+
+```javascript
+<NumberInput
+  fieldPath="cern:experiment_id"
+  label="Experiment URL"
+  placeholder="Experiment id..."
+  icon="calculator"
+  description="Unique integer id of the experiment."
+  required={true}
+/>
+```
+
+### Props
+
+- **fieldPath** `String` _required_: The path to the field e.g `cern:experiment`.
+
+- **label** `String` _required_: The label for the element's display. This is used whenever the field should be displayed e.g upload form, landing page etc.
+
+- **placeholder** `String` _required_: The placeholder for the element's display. You should fill this in with an example value.
+
+- **description** `String` _required_: The description for the element's display. You should provide useful information on how the users should fill this field in.
+
+- **icon** `String` _optional_: The optional icon that your field might have. The value should be one of the [Semantic UI icons](https://react.semantic-ui.com/elements/icon/).
+
+- **required** `Boolean` _optional_: Define if the field should be required i.e display the `*` symbol.
+
+- **disabled** `Boolean` _optional_: Define if the field should be displayed as disabled thus no input can be filled.
+
+## BooleanInput
```

Comment:
THis is a radio button and I just have internal guilts that I named it like that 😅 If you don't like it, I can rename and populate the change everywhere i.e demo site, react-invenio-forms... @ntarocco @ppanero